### DISCUSSION TITLE
关于作者问题的商榷

### DISCUSSION THREAD

**Discussion by PirateBook1** [banned-historical-archives/banned-historical-archives.github.io#831] [Q&A] [answered]
@banned-historical-archives 界定一篇文章的作者，我觉得有必要引入“广义作者”的概念：

比如，对于这篇文稿，大家公认的作者显然是王洪文和张春桥。但是我觉得，“黄文明”和“周世忠”这类的参与者也应该算进“作者”里面。再加推广一步，文稿的出版方、（再）印刷方、（再）翻印方、（再）传抄方也应该算进“作者”
![image](https://user-images.githubusercontent.com/113572850/198819052-684b80df-e583-459b-8622-d52a4560def8.png)

**Comment by PirateBook1** [comment]
各位商榷

**Comment by banned-historical-archives** [comment, marked as answer]
上面这种情况“黄文明”和“周世忠”应该加到文章的作者里面。

文章中确实应该增加一些字段存储广义的作者，包括不限于
记录员，
审核员，
翻译员，
印刷方/，
翻印/传抄方
每一种可能包含多名人员，都使用字符串数组类型。
翻印方比较特殊，用数组表示多重翻印的顺序，而不是表示多名翻印者同时翻印。例如:[日报B，杂志A, 书籍C ]

书籍的作者应该自动从其中的文稿中的作者提取。
另外书籍也应该加上这些字段，它们的内容同样从其中文稿中自动提取。
记录员，
审核员，
翻译员，
印刷方，
翻印/传抄方
（印刷方即出版社）
翻印数组中表示关联的翻印者，而不是多重翻印或者多名翻印者同时翻印

**Reply by banned-historical-archives** [reply]
翻印/传抄 性质差不多，我觉得没必要区分

**Reply by PirateBook1** [reply]
这个想法可以，排在待开发队列后吧

**Reply by banned-historical-archives** [reply]
@PirateBook1 放在标签里面？比如“印刷方：xxx”

**Reply by PirateBook1** [reply]
如果要分得详细点：
人物：xxx，事件：xxx，记录/审核：xxx，翻译：xxx，印刷：xxx，翻印/传抄：xxx

就按照目前的处理方式加进去应该就好了吧
![image](https://user-images.githubusercontent.com/113572850/220927001-0b3af5e8-fd72-4b9d-8e23-be18214b9c61.png)
![image](https://user-images.githubusercontent.com/113572850/220927152-6090ce68-685a-4fae-886b-04dfc62972e6.png)

重要的是提供方法让人工去改这些标签

**Reply by PirateBook1** [reply]
按照目前图库那边的办法，再增加几项应该就好了吧

把 tags 作为一个属性添加到文稿的对象里面去

**Reply by banned-historical-archives** [reply]
@PirateBook1 现在支持了，使用方法见ocr录入的注释。已录入的可以改books/xxx.ts

**Reply by PirateBook1** [reply]
行了，现在就剩下一个公式的问题了，看看怎么用上 LATEX
